faraday/deviceconfiguration.py

```diff
@@ -222,6 +224,10 @@ def configureDeviceConfiguration(args, deviceConfigurationConfigPath, faradayCon
         fconfig.set('BASIC', 'REDLEDTX', 1)
     if args.redledtxoff:
         fconfig.set('BASIC', 'REDLEDTX', 0)
+    if args.greenledrxon:
```

Comment:
Looks good.

@kb1lqc I just realized that all these functions don't seem to have a default or error if BOTH options are in the arguments...?